timesketch/api/experimental/resources.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2017 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Experimental parts of the Timesketch API."""
+
+
+from flask import jsonify
+from flask_restful import Resource
+from flask_login import login_required
+
+from timesketch.api.v1.resources import ResourceMixin
+from timesketch.lib.experimental.win_logins import win_logins
+from timesketch.lib.experimental.win_services import win_services
+from timesketch.lib.experimental.create_graph import create_graph
+from timesketch.lib.experimental.delete_graph import delete_graph
+
+
+class WinLoginsResource(Resource):
+    @login_required
```

Comment:
Please add a one-line docstring here and for the other methods in this file

Reply:
Ok, I will add "Handles POST request to the resource". I'm sure this will be super-helpful.

Reply:
Done.

Reply:
Thanks, this is just to stay consistent with the rest of the code base.